brd-ios/breadwallet/src/Redesign/Scenes/DeleteKYC/DeleteKYCInfo/DeleteKYCInfoInteractor.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DeleteKYCInfoInteractor.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 19/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import UIKit
+
+class DeleteKYCInfoInteractor: NSObject, Interactor, DeleteKYCInfoViewActions {
```

Comment:
nitpicking, but delete has nothing to do with KYC. Its just a requirement of apple

Reply:
i more meant like "Delete KYC Profile, Info Screen". since we are only deleting the profile, not wallet or coins and etc. maybe i can choose a better name.